tkp/accessors/lofarcasaimage.py

```diff
@@ -41,172 +62,127 @@ class LofarCasaImage(CasaImage, LofarAccessor):
         (bmaj, bmin, bpa). Will attempt to read from header if
         not supplied.
     """
+    antenna_set = None
+    ncore = nremote = nintl = 0
+    subbands = None
+    subbandwidth = None
+    subtables = {}
+    taustart_ts = None
+    tau_time = None
+    url = None
+
     def __init__(self, url, plane=0, beam=None):
         super(LofarCasaImage, self).__init__(url, plane, beam)
-
-        table = pyrap_table(self.url.encode(), ack=False)
-        subtables = open_subtables(table)
-        self._taustart_ts = parse_taustartts(subtables)
-        self._tau_time = parse_tautime(subtables)
+        self.table = pyrap_table(self.url.encode(), ack=False)
+        self.open_subtables()
+        self.parse_taustartts()
+        self.parse_tautime()
 
         # Additional, LOFAR-specific metadata
-        self._antenna_set = parse_antennaset(subtables)
-        self._ncore, self._nremote, self._nintl =  parse_stations(subtables)
-        self._subbandwidth = parse_subbandwidth(subtables)
-        self._subbands = parse_subbands(subtables)
-
-    @property
```

Comment:
I initialize these variables now in the class, and then set them in various methods called by init.

So no, this just works.